### PR TITLE
Replace chalk

### LINKED ISSUE #125: Replace chalk
I'm thinking we should replace chalk with individual `ansi-*` modules from @jonschlinkert: https://github.com/jonschlinkert?utf8=%E2%9C%93&tab=repositories&q=ansi&type=&language=

### PR DESCRIPTION
**Pull request by tdeschryver**
Closes #125

### CODE REVIEW

File: test/shared/stripAnsi.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+var ansiPattern = [
+  '[\\u001B\\u009B][[\\]()#;?]*(?:(?:(?:[a-zA-Z\\d]*(?:;[a-zA-Z\\d]*)*)?\\u0007)',
+  '(?:(?:\\d{1,4}(?:;\\d{0,4})*)?[\\dA-PRZcf-ntqry=><~]))',
+].join('|');
+var ansiRegex = new RegExp(ansiPattern, 'g');
+
+function stripAnsi(value) {
```
Comment (tdeschryver) [inline test/shared/stripAnsi.js]: Maybe it would be better to move this to `gulp-test-tools` if possible?

**Comment by phated** [comment]
I like where this is going but maybe we should combine it with the work on #63 - @sttk what do you think?

**Comment by sttk** [comment]
@phated Please merge this first. I'll address #63 later.

File: test/shared/stripAnsi.js (hunk 1 above)
Comment (sttk) [inline test/shared/stripAnsi.js]: @tdeschryver I also think it's good and welcome to send a pr to `gulp-test-tools`.

**Comment by sttk** [comment]
Does `ansi-*` support `--no-color` flag?

**Comment by tdeschryver** [comment]
@sttk it does has a [reset](https://github.com/jonschlinkert/ansi-reset) and it also has a [wrap](https://github.com/jonschlinkert/ansi-wrap). 
If this is what you're looking for :smile:

File: test/shared/stripAnsi.js (hunk 1 above)
Comment (tdeschryver) [inline test/shared/stripAnsi.js]: OK, I'll try to open a PR later today!

Comment (tdeschryver) [inline test/shared/stripAnsi.js]: Or even better, I think we should use [ansi-strip-color](https://github.com/jonschlinkert/strip-color)

**Comment by sttk** [comment]
@tdeschryver `chalk` has a feature to disable coloring by `--no-color` flag, and `gulp-cli` publishes this flag as its command line option. So we should support this flag even when using `ansi-*`.

**Comment by phated** [comment]
Looks like this still needs to support `--no-color` and there are some files that got changed (fixture `package.json` files)

**Comment by tdeschryver** [comment]
Oops, totally forgot about this 😅, I'll add during this week!

**Comment by phated** [comment]
Thanks! I'm guessing you won't need the `strip-ansi` stuff in the tests once that's supported

**Comment by tdeschryver** [comment]
Hmmm weird, seems like those fixtures which were added got changed from `CRLF` to `LF`? :confused: 
Anyways, I reverted the fixture changes.

**Comment by phated** [comment]
@tdeschryver https://github.com/doowb/ansi-colors now inlines all the colors.  This will likely be the best solution for us.  I can look into making those changes if you don't have time.

**Comment by tdeschryver** [comment]
@phated I'm currently a bit busy, if you need it right now it would be best if someone else takes this.
If you can wait a week, 2 at max, I can take it.

**Comment by phated** [comment]
Closed by #138

**Pull request closed without merging**